roles/lnls-ans-role-servweb/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+# Groups of repositories
+sirius_repos:
```

Comment:
I think it should be better to use the convention of defining variables with the role name as prefix . In this way, we avoid having to deal with a variable being redefined by another role. So, `sirius_repos`, should be `servweb_repos`

Reply:
In fact it's dangerous to "reuse" the variables name, as this is defined in the `defaults` section and this can be overridden by the command-line or other role.

Reply:
deleted